scripts/build.ts

```diff
@@ -70,7 +71,14 @@ async function main() {
   // than necessary.
   await remove(join(dirRoot, 'node_modules/fsevents'));
 
+  // Compile the `doT.js` template files for `now dev`
+  console.log();
```

Comment:
Empty console.log()? Is this just adding a new line?

Reply:
Yes